t/node/grpc-proxy.t

```diff
@@ -188,7 +188,7 @@ routes:
 GET /hello
 --- error_code: 502
 --- error_log
-upstream: "grpc://127.0.0.1:80"
+connect() failed (111: Connection refused) while connecting to upstream
```

Comment:
this change is needed as grpc proxy is done under subrequest, and the apisix custom logger (in apisix-nginx-module) overrides the original log handler of nginx and stores the reference to the original log handler in ctx.

but `ngx_stash_ctx` fails to store reference to that log handler. Thus the original log handler doesn't get executed and the `upstream: "grpc: ..."` field in error log is not present.